src/main/kotlin/net/atos/zac/signalering/SignaleringService.kt

```diff
@@ -125,11 +125,28 @@ class SignaleringService @Inject constructor(
         signalering.targettype != SignaleringTarget.USER || signalering.target != actor
 
     @Transactional(REQUIRED)
-    fun createSignalering(signalering: Signalering): Signalering {
+    fun storeSignalering(signalering: Signalering): Signalering {
         ValidationUtil.valideerObject(signalering)
-        val created = entityManager.merge(signalering)
-        eventingService.send(ScreenEventType.SIGNALERINGEN.updated(created))
-        return created
+
+        val signaleringToStore = findSignalering(signalering)?.apply {
+            LOG.info("Replacing $this timestamp $tijdstip with ${signalering.tijdstip}")
+            tijdstip = signalering.tijdstip
+        } ?: signalering
+
+        return entityManager.merge(signaleringToStore).also {
+            eventingService.send(ScreenEventType.SIGNALERINGEN.updated(it))
```

Comment:
In DB we do not have new signalering, but we still send screen event.

This means that in DB we have 1 event, while FE will list 2 with the real-time update. After refresh of the page the second one will disappear.

@infonl-marcel @edgarvonk I don't think this is correct. Perhaps we need to skip sending the event altogether?

Reply:
I don't know. Sounds like a functional question? Maybe make a proposal and ask Karin? What do users expect to happen?

Reply:
Moved the discussion to Slack